src/app/information/information/information.component.html

```diff
@@ -1,23 +1,24 @@
 <section class="section">
   <div class="container">
     <div fxLayout="row" fxLayoutAlign="center center">
-      <img src="../../assets/cce-logo.png">
+      <img src="../../assets/cce-logo.png" />
     </div>
     <div class="vertical-rhythm" fxLayout="column" fxLayoutAlign="center center">
       <div fxLayout="column">
         <h3 class="text-primary text-center">
-          We Just need a little bit more<br> information...
+          We Just need a little bit more<br />
```

Comment:
We just (Just is capitalized)

Reply:
stupid autoformatter...I didn't change that crap, but fine...